solutions/src/main/java/ru/sberbank/school/task07/WordsFrequencyImpl.java

```diff
@@ -0,0 +1,28 @@
+package ru.sberbank.school.task07;
+
+import lombok.NonNull;
+
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class WordsFrequencyImpl implements WordFrequency<Map<String, Integer>> {
+
+    private FileParser parser;
+
+    public WordsFrequencyImpl(@NonNull FileParser parser) {
+        this.parser = parser;
+    }
+
+    @Override
+    public Map<String, Integer> countWords(String pathToFile) throws FileNotFoundException {
+
+        Map<String, Integer> map = new HashMap<>();
+
+        parser.parse(pathToFile).
```

Comment:
&#39;.&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1283204456>*